shared/cfg/guest-os/Linux/Fedora/23.ppc64le.cfg

```diff
@@ -0,0 +1,18 @@
+- 23.ppc64le:
+    image_name = images/f23-64
+    vm_arch_name = ppc64le
+    os_variant = fedora23
+    no unattended_install..floppy_ks
+    boot_path = ppc/ppc64
+    unattended_install, svirt_install:
+        kernel_params = 'repo=cdrom:/dev/disk/by-label/Fedora-Server-23-ppc64le'
+        kernel_params += ' ks=cdrom nicdelay=60 console=hvc0 console=ttyS0 serial'
+        unattended_file = unattended/Fedora-23.ks
+        cdrom_unattended = images/f23-64/ks.iso
+        kernel = images/f23-64/vmlinuz
```

Comment:
I don't see the change here, did you commit the change? All the `f23-64` needs to point to different location, RHE uses `$name-ppc64le` to avoid clashes with x86_64. Would you please replace all `f23-64` with `f23-ppc64le` and commit it here?